desc/compute/_metric.py

```diff
@@ -1400,9 +1400,113 @@ def _g_sup_rr_r(params, transforms, profiles, data, **kwargs):
+    units="m^-2",
+    units_long="inverse square meters",
+    description="Radial/Radial element of contravariant metric tensor, "
+    + "first toroidal derivative",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["e^rho", "e^rho_z"],
+)
+def _g_sup_rr_z(params, transforms, profiles, data, **kwargs):
+    data["g^rr_z"] = 2 * dot(data["e^rho_z"], data["e^rho"])
+    return data
+
+
+@register_compute_fun(
+    name="g^rr_zz",
+    label="\\partial_{\\zeta \\zeta} g^{rr}",
+    units="m^-2",
+    units_long="None",
+    description="Double zeta derivative of the flux expansion term",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=[
+        "e^rho",
+        "e^rho_z",
+        "e^rho_zz",
+    ],
+)
+def _g_sup_rr_sub_zz(params, transforms, profiles, data, **kwargs):
+    data["g^rr_zz"] = 2 * (
+        dot(data["e^rho_zz"], data["e^rho"]) + dot(data["e^rho_z"], data["e^rho_z"])
+    )
+    return data
+
+
+@register_compute_fun(
+    name="g^rr_t",
+    label="\\partial_{\\theta} g^{\\rho}{\\rho}",
+    units="m^-2",
+    units_long="inverse square meters",
+    description="Radial/Radial element of contravariant metric tensor, "
+    + "first poloidal derivative",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["e^rho", "e^rho_t"],
+)
+def _g_sup_rr_t(params, transforms, profiles, data, **kwargs):
+    data["g^rr_t"] = 2 * dot(data["e^rho_t"], data["e^rho"])
+    return data
+
+
+@register_compute_fun(
+    name="g^rr_tt",
+    label="\\partial_{\\theta \\theta} g^{\\rho}{\\rho}",
+    units="m^-2",
+    units_long="inverse square meters",
+    description="Radial/Radial element of contravariant metric tensor, "
+    + "second poloidal derivative",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["e^rho", "e^rho_t", "e^rho_tt"],
+)
+def _g_sup_rr_tt(params, transforms, profiles, data, **kwargs):
+    data["g^rr_tt"] = 2 * (
+        dot(data["e^rho_tt"], data["e^rho"]) + dot(data["e^rho_t"], data["e^rho_t"])
+    )
+    return data
+
+
+@register_compute_fun(
+    name="g^rr_tz",
+    label="\\partial_{\\theta \\zeta} g^{\\rho}{\\rho}",
+    units="m^-2",
+    units_long="inverse square meters",
+    description="Radial/Radial element of contravariant metric tensor, "
+    + "mixed derivative",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["e^rho", "e^rho_t", "e^rho_z", "e^rho_tz"],
+)
+def _g_sup_rr_tz(params, transforms, profiles, data, **kwargs):
+    data["g^rr_tz"] = 2 * (
+        dot(data["e^rho_tz"], data["e^rho"]) + dot(data["e^rho_t"], data["e^rho_z"])
+    )
+    return data
+
+
 @register_compute_fun(
     name="g^rt_r",
-    label="\\partial_{\\rho} g^{\\rho \\theta}",
+    label="\\partial_{\\rho} g^{\\rho}{\\theta}",
```

Comment:
this will format incorrectly, we want both rho and theta in the superscript:
$g^{\\rho \\theta}$

not 

$g^{\\rho}{\\theta}$

Reply:
Resolved